### ISSUE TITLE
SunEditor: 3.X Upgrade and Dark Theme

### ISSUE THREAD

**Issue by melloware** [primefaces-extensions/primefaces-extensions#1138]
### Describe the feature you would like to see added

Theme switching https://github.com/JiHong88/suneditor/issues/978

Once that issue is resolved we should update it to add an "auto" mode to detect theme and switch to dark like we do with other components.

**Comment by AreebAfzal** [comment]
Do we have dark theme in suneditor now?

**Comment by jepsar** [comment]
@AreebAfzal as you can see, this issue is still open.

**Comment by melloware** [comment]
@AreebAfzal apparently the SunEditor developer is working on a Dark Theme for his 3.0 release.

**Comment by melloware** [comment]
SunEditor adding theming in a 3.0.alpha.  I think I will wait until the 3.0 final release though.

**Comment by Mahdibayat** [comment]
I Need To 🚶‍♂️

**Comment by last-resort** [comment]
SunEditor 3.0 was released yesterday.

**Comment by melloware** [comment]
@last-resort yep!  Looks like it has some significant changes we will have to work through but this is a good thing!
